frontendIcons/src/main/java/icons/MacheteIcons.java

```diff
@@ -6,6 +6,26 @@
 
 public final class MacheteIcons {
   private MacheteIcons() {}
-  public static final Icon ICON = IconLoader.getIcon("/macheteFileIcon.svg");
+
```

Comment:
While you're here, pls drop a comment why this must be `icons` and not e.g. `com.virtuslab.gitmachete.icons` package

Reply:
@micpiotrowski

Reply:
okay, I got it
https://www.jetbrains.org/intellij/sdk/docs/reference_guide/work_with_icons_and_images.html